main.typ

```diff
@@ -17,6 +17,8 @@
 #pagebreak()
 #include "system-overview.typ"
 #pagebreak()
+#include "top-level-functions.typ"
+#pagebreak()
```

Comment:
Would it make more sense to have these the other way around?
```suggestion
#include "top-level-functions.typ"
#pagebreak()
#include "system-overview.typ"
#pagebreak()
```

Reply:
That's fair.